.vscode/sftp.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file needs to be deleted, and your `.vscode/**` folder need to be added to your machines global git ignore file